### PR TITLE
Update documation drawer look & feel

### PR DESCRIPTION
**Pull request by OskarDamkjaer**
- Adds proper types
- Adds and shows common snippets, runnable on click
- Adds hover effects (icon and background color)
- Sets new default color for links (!important) needed until we remove dependency on semantic-ui

Old look:
image

New look:
image

Try it yourself on this url:  http://refresh_docs_drawer.surge.sh

### CODE REVIEW

File: src/browser/modules/Sidebar/styled.tsx [hunk 1]
```diff
@@ -45,33 +43,68 @@ export const StyledSettingTextInput: any = styled.input`
   width: 192px;
 `
 
-export const StyledHelpLink = styled.a``
+export const StyledHelpLink = styled.a`
+  cursor: pointer;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  color: #68bdf4;
+
```
Comment (HerrEmil) [inline src/browser/modules/Sidebar/styled.tsx:51]: active decoration none

File: src/browser/modules/Sidebar/styled.tsx [hunk 2]
```diff
@@ -45,33 +43,68 @@ export const StyledSettingTextInput: any = styled.input`
...
+  }
+`
 export const StyledHelpItem = styled.li`
   list-style-type: none;
   margin: 8px 0 0 0;
 `
 
-const StyledDocumentText = styled.a`
+export const StyledCommandListItem = styled.li`
+  margin: 0px -39px 0 -24px; // Get full width background hover effect
```
Comment (HerrEmil) [inline src/browser/modules/Sidebar/styled.tsx:67]: Try overwriting padding

**Review by HerrEmil — APPROVED** [review]

File: src/browser/modules/Sidebar/styled.tsx (hunk 2 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Sidebar/styled.tsx]: Done!

File: src/browser/modules/Sidebar/styled.tsx (hunk 1 above)
Comment (OskarDamkjaer) [reply src/browser/modules/Sidebar/styled.tsx:50]: 👍

**Pull request merged**